Detections/MultipleDataSources/MalformedUserAgents.yaml

```diff
@@ -30,13 +33,23 @@ query: |
   (OfficeActivity | where TimeGenerated >= ago(endtime) | where UserAgent != ""),
```

Comment:
Since we are doing a union between several tables, for all tables in the query, we should consider projecting only the most desired columns and we should also consider mapping similar column names to the same name to reduce the number of columns in the result.

Reply:
Done. Thanks Shain